### DISCUSSION TITLE
Keep details from basemap on zoom levels > 11

### DISCUSSION THREAD

**Discussion by naoliv** [osmandapp/OsmAnd#13544] [Ideas]
It seems that we have roads and some other details in the mini map from zoom level `6` to `8`.

Couldn't we keep the details further?

At higher zoom levels, viewing a roughly detailed area in the map should be better than no data at all (on regions where we don't have a local map downloaded).

**Comment by scaidermern** [comment]
I think this is essentially the same as #7999.

**Comment by naoliv** [comment]
It's very similar, indeed.

But from #7999 it seems that a map with more details/data is requested while I was thinking that osmand should keep displaying the lower quality data even on higher zoom levels.
ie, if we have data up to zoom level `8`, simply keep zooming in this data (since a lower resolution/rough data should be better than no data at all).

**Comment by scaidermern** [comment]
> But from #7999 it seems that a map with more details/data is requested while I was thinking that osmand should keep displaying the lower quality data even on higher zoom levels.

That's what I suggested in https://github.com/osmandapp/Osmand/issues/7904#issuecomment-553014542 but the developers decided otherwise :/

**Comment by vshcherb** [comment]
duplicate of Detailed basemap https://github.com/osmandapp/Osmand/issues/7999

**Comment by naoliv** [comment]
This is a different issue and it's not fixed with the detailed world overview map.

Try to zoom further than zoom level 11 and see that there aren't any objects/roads/details/anything when we have only the detailed world overview map enabled.

OsmAnd should keep displaying the objects from the detailed world overview map even when we don't have a regional map enabled.

**Comment by tradavyvy** [comment]
I think this problem deserves better then a nice to have.
It is always better to display data you already have then display nothing.
And for those who think why not download the map if you want to drive/walk/be somewhere:
I miss this info when i add "random" favorites while reading a book or watching tv or talking to someone. I might hear about a beautifull temple, waterfall or nice restaurant.
I travel extensively all over the world, but do not have the memmory to load alll available maps.
So while adding favorites it would be nice to have some (zoomed in) reference to where they are.

Here is what happens when zooming:

![Screenshot_20220103-073327_OsmAnd+](https://user-images.githubusercontent.com/71978409/147905336-6c097637-518d-42ea-9413-c10988f3e51a.jpg)

![Screenshot_20220103-073350_OsmAnd+](https://user-images.githubusercontent.com/71978409/147905360-133fc05c-596f-4428-a1ce-b38c54e86657.jpg)

![Screenshot_20220103-073400_OsmAnd+](https://user-images.githubusercontent.com/71978409/147905371-bcbf76bf-f75e-4aeb-9f5d-9480d860839a.jpg)

All references disapear.

**Comment by vshcherb** [comment]
It's done on purpose to not overscale bad artifacts from basemap, you could always win +1 / 1.5 zoom by using zoom magnifier

**Comment by naoliv** [comment]
> It's done on purpose to not overscale bad artifacts from basemap, you could always win +1 / 1.5 zoom by using zoom magnifier

Isn't badly drawn data better than no data displayed at all?

**Reply by vshcherb** [reply]
Convert to discussion to have - threads conversation.

We've switched to approach, yes points information is better than everything is displayed. It has artifacts that don't reflect reality anymore. It's possible to readjust basemap so it continues display details cause this is a data driven solution (not hardcoded in the code) .
